### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by ArianaShackelford**
### Summary

- Setup unit testing within `src/components/ConnectionsViewTabs/AddDocumentButtonsGroup`.
- Setup `jest` configuration to test on Android and iOS.

### Details

- Spend an unreasonable amount of time _debugging_ `jest` and `jest-expo`... Aren't these supposed to make testing _easier_?

- Perform unit testing for: 

    - functions:

        - `convertFileToMedia`
        - `parseBaseNameFromPath`
        - `parseBaseNameFromUri`
        - `parseDirNameFromPath`
        - `parseBirNameFromUri`

    - components:

        - `PickFileButton`
        - `PickPhotoButton`
        - `TakePhotoButton`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +45,7 @@
     "react-native-reanimated": "~1.4.0",
     "react-native-screens": "2.0.0-alpha.12",
     "react-native-switch-toggle": "^1.1.2",
+    "react-native-testing-library": "^1.12.0",
```
Comment (jason-glassbrook) [inline package.json]: should move to dev dependencies

File: package.json [hunk 2]
```diff
@@ -52,6 +54,7 @@
     "react-navigation-stack": "^1.10.3",
     "react-navigation-tabs": "^2.5.6",
     "react-redux": "^7.1.0",
+    "react-test-renderer": "^16.13.0",
```
Comment (jason-glassbrook) [inline package.json]: should move to dev dependencies

File: package.json (hunk 1 above)
Comment (Sara-DLC) [reply package.json]: will do

File: package.json (hunk 2 above)
Comment (Sara-DLC) [reply package.json]: will do

**Pull request merged**